helix-loader/src/lib.rs

```diff
@@ -122,10 +121,11 @@ pub fn cache_dir() -> PathBuf {
 }
 
 pub fn config_file() -> PathBuf {
-    CONFIG_FILE
-        .get()
-        .map(|path| path.to_path_buf())
-        .unwrap_or_else(|| config_dir().join("config.toml"))
+    CONFIG_FILE.get().map(|path| path.to_path_buf()).unwrap()
+}
+
+pub fn log_file() -> PathBuf {
+    LOG_FILE.get().map(|path| path.to_path_buf()).unwrap()
```

Comment:
I was about to write the following here (and the equivalent for the `config_file()`)

```rust
unwrap_or_else(|| {
   initialize_log_file(None)
   log_file()
})
```

But then I really think we should `unwrap()` and potentially fail. A failure here means we have seriously broken the initialization steps in the `main.rs`, it seems less misleading than re-replicating the initialization of the cell in two places.